### DISCUSSION TITLE
variables

### DISCUSSION THREAD

**Discussion by clubeplus** [nocodb/nocodb#4332] [Q&A]
Where can I edit the variables, I can't find the location, I installed it via docker

**Comment by dstala** [comment]
Do you mean `database connection parameters`? Currently - you will not be able to edit configured parameters.

**Comment by o1lab** [comment]
If you are new to docker and setting env variables - you should do google search [for this](https://www.google.com/search?q=setting+environment+variables+for+docker+container&oq=setting+environment+variables+for+docker&aqs=chrome.0.0i512j69i57j0i22i30l7j0i390.4545j0j7&sourceid=chrome&ie=UTF-8).
